### PR TITLE
ContainerRuntime: Remove the contents property of batchBegin/batchEnd's "op" event arg

### LINKED ISSUE #22846: ContainerRuntime: Remove the contents property of batchBegin/batchEnd's "op" event arg
## Deprecated API

ContainerRuntime's 'batchBegin'/'batchEnd' events: Removing the `contents` property on event arg `op`

**Context**

The 'batchBegin'/'batchEnd' events on ContainerRuntime indicate when a batch is beginning/finishing being processed.
The `contents` property on there is not useful or relevant when reasoning over incoming changes at the batch level. So, it has been removed from the `op` event arg.

https://github.com/microsoft/FluidFramework/pull/22791

### PR DESCRIPTION
**Pull request by markfields**
## Description

Fixes [AB#19784](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/19784)

ContainerRuntime's 'batchBegin'/'batchEnd' events: Removing the `contents` property on event arg `op`

The 'batchBegin'/'batchEnd' events on ContainerRuntime indicate when a batch is beginning/finishing being processed.
The `contents` property on there is not useful or relevant when reasoning over incoming changes at the batch level.
So it has been removed from the `op` event arg.

## Breaking Changes

Yes this is a breaking change.  See changeset.

### CODE REVIEW

**Comment by markfields** [comment]
Converting to Draft until `main` is open for breaking changes

**Comment by markfields** [comment]
I'll be OOF until 11/14.  I'll plan to bring this in for 2.10.0 at that point if it's not too late (unless someone decides to merge this for me while I'm OOF)

**Review by kian-thompson — APPROVED** [review]

**Comment by markfields** [comment]
@kian-thompson feel free to merge this if the time is right for breaking changes. Not paying attention broadly, just saw the notification that you approved it.

**Review by jatgarg — APPROVED** [review]

File: .changeset/fruity-sites-boil.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+---
+---
+"section": other
+---
+
+ContainerRuntime's 'batchBegin'/'batchEnd' events: Removing the `contents` property on event arg `op`
+
+The 'batchBegin'/'batchEnd' events on ContainerRuntime indicate when a batch is beginning/finishing being processed.
+The `contents` property on there is not useful or relevant when reasoning over incoming changes at the batch level.
+So it has been removed from the `op` event arg.
```
Comment (tylerbutler) [inline .changeset/fruity-sites-boil.md:12]:
```suggestion
Changes to the batchBegin and batchEnd events on ContainerRuntime

The 'batchBegin'/'batchEnd' events on ContainerRuntime indicate when a batch is beginning or finishing being processed. The `contents` property on the event argument `op` is not useful or relevant when reasoning over incoming changes at the batch level. Accordingly, it has been removed from the `op` event argument.
```

**Review by tylerbutler — APPROVED** [review]
reworded the changeset a bit - approving for docs.

File: .changeset/fruity-sites-boil.md (hunk 1 above)
Comment (markfields) [reply .changeset/fruity-sites-boil.md:12]: @tylerbutler you'll have to make a follow-up PR, Kian had auto-merge on. I'm still OOF just saw the notification

Comment (tylerbutler) [reply .changeset/fruity-sites-boil.md:12]: No worries, I'll chat with Kian about it.

Comment (kian-thompson) [reply .changeset/fruity-sites-boil.md:12]: https://github.com/microsoft/FluidFramework/pull/23015

**Pull request merged**